apps/server/package.json

```diff
@@ -22,11 +22,15 @@
 	"homepage": "https://github.com/overture-stack/lectern#readme",
 	"devDependencies": {
 		"@types/body-parser": "^1.19.5",
+		"@types/chai": "^4.3.16",
+		"@types/chai-http": "^4.2.4",
+		"@types/cors": "^2.8.18",
 		"@types/errorhandler": "0.0.32",
 		"@types/express": "^4.17.21",
```

Comment:
upgraded `express` package but not it's types

Reply:
This helped me spot an underlying type issue, thank you.